json_encoder.go

```diff
@@ -95,13 +95,17 @@ func (enc *jsonEncoder) Free() {
 	jsonPool.Put(enc)
 }
 
+func (enc *jsonEncoder) wrapString(val string) {
```

Comment:
Perhaps `addAndQuoteString`? I'd expect `wrapString` to return the wrapped value.